documentation/airport-format.md

```diff
@@ -1,309 +1,551 @@
+"07L": ["WASTE", ["BAKRR", "A70+|S250-"], "COMPS"]
+```
+These definitions can be used within any `Entry`, `Body` or `Exit` segment of a standardRoute.
+
+
+### sids
+_All properties in this section are required for each route definition_
+
+```javascript
+"sids": {
+    "COWBY6": {
+        "icao": "COWBY6",
+        "name": "Cowboy Six",
+        "suffix": {"1L":"", "1R":"", "28L":"", "28R":""},
+        "rwy": {
+            "01L": ["_NAPSE068", "NAPSE", ["RIOOS", "A130+"], "COMPS"],
+            "01R": ["_NAPSE068", "NAPSE", ["RIOOS", "A130+"], "COMPS"],
+            "07L": ["WASTE", ["BAKRR", "A70"], "COMPS"],
+            "07R": ["JESJI", ["BAKRR", "A70"], "COMPS"],
+            "19L": ["FIXIX", ["ROPPR", "A70"], ["CEASR", "A80+"], ["HITME", "A110+"]],
+            "19R": ["JAKER", ["ROPPR", "A70"], ["CEASR", "A80+"], ["HITME", "A110+"]],
+            "25L": ["PIRMD", ["ROPPR", "A70"], ["CEASR", "A80+"], ["HITME", "A110+"]],
+            "25R": ["RBELL", ["ROPPR", "A70"], ["CEASR", "A80+"], ["HITME", "A110+"]]
+        },
+        "body": ["COWBY"],
+        "exitPoints": {
+            "DRK": ["NAVHO", "DRK"],
+            "GUP": [["MOSBI", "A150+"], "GUP"],
+            "INW": [["CUTRO", "A150+"], "INW"]
         },
-      "body": ["EUGEN", "SHOEY"],   // (optional) If there is a very long series of fixes in a SID, it may be
-                                    // helpful to put some of it here, while all segments follow the same path.
-      "exitPoints": {     // (optional) Defines exitPoints for a given SID. Common for FAA-style (USA) SIDs. If not needed (like in Europe), leave this part out.
-          "SNS": ["SNS"], // defines the "OFFSH9.SNS" transition as being a single fix, "SNS". Is often a list instead.
-          "BSR": ["BSR"]  // Note that this connects to the end of previous sections, so an example route: SEPDY->ZUPAX->EUGEN->SHOEY->BSR
-      },
-      "draw": [["SEPDY","ZUPAX"], ["SENZY","ZUPAX","EUGEN","SHOEY*"], ["SHOEY","SNS*"], ["SHOEY","BSR*"]]
-        // (req) This "draw" section is what defines how the SID is to be drawn on the scope in blue.
-        // The array contains multiple arrays that are a series of points to draw fixes between.
-        // In this case, SEPDY->ZUPAX, SENZY->ZUPAX->EUGEN->SHOEY, SHOEY->SNS, SHOEY->BSR are the lines drawn.
-        // Additionally, you'll notice three asterisks ('*'). This is an optional flag that, if invoked for "FIXXX"
-        // will tell canvas.js to write "OFFSH9.FIXXX" next to FIXXX on the scope. If no such flags are present,
-        // then the ICAO identifier for the SID will be drawn at the last point of the "draw" array. For european-
-        // style SIDs, where they always end at the fix for which the SID is named, don't use the flags. But if your SID
-        // has transitions, like in the N/S Americas, United Kingdom, etc, be sure to flag all the transition fixes.
+        "draw": [
+            ["ROPPR", "CEASR", "HITME", "COWBY", "MOSBI", "GUP*"],
```

Comment:
Used to determine the blue lines (for SIDs) and (albeit not yet implemeneted) red lines (for STARs) from fix to fix along the procedures. Each element within the `draw` array is a "line" to draw, containing the individual fixes to draw lines between.

As for the asterisks: 
>Additionally, you'll notice three asterisks ('*'). This is an optional flag that, if invoked for "FIXXX" will tell canvas.js to write "OFFSH9.FIXXX" next to FIXXX on the scope. If no such flags are present, then the ICAO identifier for the SID will be drawn at the last point of the "draw" array. For european-style SIDs, where they always end at the fix for which the SID is named, don't use the flags. But if your SID has transitions, like in the N/S Americas, United Kingdom, etc, be sure to flag all the transition fixes.